addons/compat_spe/compat_spe_wardrobe/ACE_wardrobe_uniforms.hpp

```diff
@@ -0,0 +1,128 @@
+// UNIFORM_SLEEVES(variantSleevesDown,variantSleevesUp);
+// UNIFORM_GLOVES(variantGlovesOn,variantGlovesOff);
+
+UNIFORM_GLOVES(U_SPE_FSJ_mk2,U_SPE_FSJ_mk2_gloves);
+
```

Comment:
In my opinion, because you have added gloved variants of US uniforms to the wardrobe, the `U_SPE_CIV_Citizen_X` should be able to be turned into `U_SPE_CIV_Citizen_X_tie` (adding/removing a necktie).